core-services/result-aggregator/result-aggregator.yaml

```diff
@@ -86,6 +86,8 @@ items:
           image: result-aggregator:latest
           args:
           - -log-level=debug
+          - -username=ci
+          - -word=/etc/report/password.txt
```

Comment:
almost

Reply:
some editor bullshit got this, what the hell